src/components/Sidebar.tsx

```diff
@@ -1,56 +1,86 @@
+    setSidebarVisible((visible) => !visible);
+  };
+
   return (
-    <div
-      className="\ top-0 flex h-screen w-[160px] flex-col justify-start gap-20
-                    border-r border-darker-tan bg-med-tan py-12"
-    >
-      <div className="flex flex-col place-items-center p-4">
-        <Link href="/">
-          <h1 className="z-10 text-center font-serif text-2xl duration-150 hover:-translate-y-0.5">
+    <div className="sticky top-0 flex h-screen w-full flex-col items-center justify-between bg-med-tan px-11 py-20">
+      <div className="w-full">
+        <Link href="/public/">
+          <h1 className="z-10 mb-8 text-center font-serif text-3xl duration-150 hover:-translate-y-0.5">
             The Legacy Project
           </h1>
         </Link>
-
-        <span className="absolute left-3 top-[48px] aspect-square h-[60px] rotate-45 opacity-60">
-          <Image
-            src="/landing/pink-flower.png"
-            layout="fill"
-            objectFit="cover"
-            // TODO(nickbar01234) - Write a more descriptive alt
-            alt="Pink flower"
-          />
-        </span>
-
-        <span className="absolute left-[88px] top-[70px] aspect-square h-[60px] opacity-40">
-          <Image
-            src="/landing/yellow-flower.png"
-            layout="fill"
-            objectFit="cover"
-            // TODO(nickbar01234) - Write a more descriptive alt
-            alt="Yellow flower"
-          />
-        </span>
+        <div className="flex w-full flex-col space-y-6">
+          {buttons.map((data) => (
+            <Link key={data.name} href={data.link}>
+              <SidebarItem label={data.name} iconName={data.icon} />
+            </Link>
+          ))}
+        </div>
       </div>
 
-      <div className="flex w-full flex-col border border-darker-tan bg-dark-tan">
-        {/* TODO(nickbar01234) - Point this to something... */}
-        <Link href="/home">
-          <SidebarItem label="Home" />
-        </Link>
-        <button onClick={() => signOut({ callbackUrl: "/" })}>
-          <SidebarItem label="Sign Out" />
+      <div className="w-full">
+        {/* TODO(nickbar01234) - Render university name */}
+        {/* <div className="text-md flex pb-2 pt-20 text-left font-serif text-xxs text-dark-gray">
+          <div className="w-4">
+            <FontAwesomeIcon icon={faCity} />
+          </div>
+          University
+        </div>
+        <div className="text-md flex w-full text-left font-serif text-xs">
+          Tufts University
+        </div> */}
+        <hr className="my-6 h-px w-full rounded border-0 bg-black" />
+        <div className="mb-1 flex w-full text-left font-serif text-lg font-bold">
+          {user.name ?? ""}
+        </div>
+        <div className="flex w-full pb-6 pt-0 text-left font-serif text-sm text-med-gray">
+          {user.role[0] + user.role.toLowerCase().slice(1)}
+        </div>
+        <button onClick={() => signOut({ callbackUrl: "/public/" })}>
```

Comment:
Added a new redirected url since `/` doesn't exist anymore.

Reply:
Thank you! Sounds good